qulice-findbugs/src/main/java/com/qulice/findbugs/Wrap.java

```diff
@@ -87,7 +92,7 @@ public static void run(final String... args) {
         }
         final FindBugs2 findbugs = new FindBugs2();
         findbugs.setProject(project);
-        final Collection<String> ignored = Collections2.filter(
+        final Collection<String> ignored = //Collections2.filter(
```

Comment:
@dskalenko we do not comment out code - you can put just snippet or commit ID to puzzle instead